cypress/integration/unit_tests/checkbox_test.js

```diff
@@ -176,6 +178,103 @@ describe('Unit test for toggleLayerOn', () => {
     });
   });
 
+  it('fails to load a layer', () => {
+    setUpLayerFailure().then(
+        () => addLayer(
+            {
+              'ee-name': 'asset/does/not/exist',
+              'asset-type': LayerType.FEATURE_COLLECTION,
+              'display-name': 'asset1',
+              'display-on-load': false,
+              'color-function': colorProperties,
+              'index': 3,
+            },
+            null));
+    assertLayerFailure();
+  });
+
+  it('fails to toggle on a layer', () => {
+    const layer = {
+      'ee-name': 'asset/does/not/exist',
+      'asset-type': LayerType.FEATURE_COLLECTION,
+      'display-name': 'asset1',
+      'display-on-load': false,
+      'color-function': colorProperties,
+      'index': failureLayerIndex,
+    };
+    addNullLayer(layer);
+    setUpLayerFailure().then(
+        () => toggleLayerOn(
+            {
+              'ee-name': 'asset/does/not/exist',
+              'asset-type': LayerType.FEATURE_COLLECTION,
+              'display-name': 'asset1',
+              'display-on-load': false,
+              'color-function': colorProperties,
```

Comment:
why can't just user layer here? probably same answer, but can also reuse between tests?

Reply:
Not sure I understand the question :) the asset type changes between two of the tests, didn't think worth extracting just for the remaining two tests for the second part.

Reply:
Ah gotcha, missed some of the subtle differences. sgtm.